gems/pending/util/miq_logger_processor.rb

```diff
@@ -83,8 +83,8 @@ def self.to_png(data, options = {})
   end
 
   def self.csv_to_png(filename, options = {})
-    require 'fastercsv'
-    to_png(FasterCSV.read(filename), options)
+    require 'csv'
+    to_png(CSV.read(filename), options)
```

Comment:
@agrare or you can change it to this so you can test just the `read_csv` method:

```ruby
def self.csv_to_png(filename, options = {})
  to_png(read_csv(filename), options)
end

def self.read_csv(filename)
  require 'csv'
  CSV.read(filename)
end
```